quodlibet/quodlibet/qltk/quodlibetwindow.py

```diff
@@ -129,34 +122,36 @@ def single(self):
         When `repeat` is True the current song will be replayed.
         """
 
-        return self._order.get_active_name() == "onesong"
+        # FIXME: Add OneSong repeat mode etc
+        return False
 
     @single.setter
     def single(self, value):
         if value and not self.single:
-            self._order.set_active_by_name("onesong")
```

Comment:
You can test through ``mpris-remote repeat true`` btw

or using mpdroid on android (in the top right corner menu you can enable single mode)